web/thesauruses/c/17/io.json

```diff
@@ -0,0 +1,278 @@
+            },
+       "clear_console_cutput": {
+            "code": [
+                "#include<stdio.h>",
+                "#include<stdlib.h>",
+                "int main(){",
+                "system(\"cls\");",
+                "printf(\"hello\");",
+                "return 0;",
+                "}"
+            ], 
+            "comment": "system() is defined in stdlib.h",
+            "name": "Clear the console output"
+        },
+        "change_console_background_color":{
+            "not-implemented":"true",
+            "name":"change the background color of the console"
+        },
+        "change_console_text_color":{
+            "not-implemented":"true",
+            "name":"change the text color of the console"
+        },
+        "file_functions_lib": {
+            "not-implemented":"true",
+            "name":"Library containing file IO functions"
+        },
+        "list_directory":{
+            "not-implemented":"true",
+            "name":"list of the items in the current directory"
+        },
+        "read_directory_info":{
+            "not-implemented":"true",
+            "name":"Get information about the current directory (number of items, size, etc)"
+        },
+        "create_directory":{
+            "not-implemented":"true",
+            "name":"create a directory"
+        },
+        "delete_directory":{
+            "not-implemented":"true",
+            "name":"delete a directory"
+        },
+        "rename_directory":{
+            "not-implemented":"true",
+            "name":"rename a directory"
+        },
+        "move_directory":{
+            "not-implemented":"true",
+            "name":"move a directory to another location"
+        },
+        "update_directory_permissions":{
+            "not-implemented":"true",
+            "name":"Update the permissions a user, group, or application has to a directory"
+        },
+      "read_line_of_file": {
+        "not-implemented":"true",
+        "name":"Read a single line from a file"
+      },
+      "read_lines_of_file":{
+        "not-implemented":"true",
+        "name":"Read all the lines in a file"
+    },
+      "read_file_stream":{
+        "not-implemented":"true",
+        "name":"Read the file into a stream"
+      }, 
+        "create_file": {
+            "code": "FILE *fopen( const char * filename, const char * mode );",
+            "name": "Create a file"
+        },
+        "delete_file": {
+            "code": "int remove ( const char * filename );",
+            "comment": "returns 0 if file is deleted",
+            "name": "Delete a file"
+        },
+        "update_file": {
+            "code": "FILE *fopen( const char * filename, const char * mode );",
+            "comment": "different file modes are available",
+            "name": "Update a file"
+        },
+        "move_file":{
+            "not-implemented":"true",
+            "name":"move file to a new location"
+        },
```

Comment:
Wellll...  there's file functions in C. (See https://en.cppreference.com/w/c/io )